add better description for the fluent-bit container exclude (#15109)
kyma-project/kyma: docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

        - fluent-bit
```

> It is recommended to exclude the logs of the Fluent Bit container to avoid problems with recursive logs in case Fluent Bit will print an error per processed log line which then gets collected and processed again.

Reviewer: ```suggestion
It might happen that Fluent Bit prints an error per processed log line that is then collected and re-processed.
To avoid problems with such recursive logs, it is recommended that you exclude the logs of the Fluent Bit container. 
```

This sentence is very long. How about this instead?

| input.application.excludeNamespaces | []string | List of Namespaces to exclude during log collection from all Namespaces (mutually exclusive with `namespaces`) |
| input.application.containers | []string | List of containers to collect from (mutually exclusive with `excludeContainers`) |
| input.application.excludeContainers | []string | List of containers to exclude (mutually exclusive with `containers`) |

> | input.application.includeSystemNamespaces | boolean | If you specify neither `namespaces` nor `excludeNamespaces`, you can set `includeSystemNamespaces` to include all system Namespaces without listing them manually. Defaults to `false`. If enabled, it is recommended to exclude the logs from the Fluent Bit container to avoid problems with recursive logs, see also [Create a LogPipeline Input](#step-2-create-an-input) |

Reviewer: ```suggestion
| input.application.includeSystemNamespaces | boolean | If you specify neither `namespaces` nor `excludeNamespaces`, you can set `includeSystemNamespaces` to include all system Namespaces without listing them manually. Defaults to `false`. If enabled, it is recommended to exclude the logs from the Fluent Bit container to avoid problems with recursive logs. See also [Create a LogPipeline Input](#step-2-create-an-input). |
```
